srv/dte-energy-admin-service.cds

```diff
@@ -1,10 +1,18 @@
 using {dteConsentApp as db} from '../db/schema';
 
 @path : '/admin/service'
+@impl : './handler/admin-handler.js'
 service DTEEnergyAdminPortal {
 
-   entity ApplicationDetail as projection on db.ApplicationDetail;
-
+  entity ApplicationDetail as projection on db.ApplicationDetail actions{
+    function GenerateUrl() returns String;
+    action UpdateAppDetail(AppId: String, NoOfTenants: Integer, AppStatus: String) returns String;
+  };
+  
+  entity ApplicationConsent as projection on db.ApplicationConsent;
```

Comment:
Update ConsentStatus Action Missing.

Reply:
The updation of fields can be achieved by the OData model configuratioin. So this is no more needed here.